hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/main/java/org/apache/hadoop/yarn/util/DockerClientConfigHandler.java

```diff
@@ -103,7 +103,7 @@ public static Credentials readCredentialsFromConfigFile(Path configFile,
 
```

Comment:
In L99-101:
* ObjectMapper instance should be reused.
* Two methods are deprecated.
Should be fixed in a separate jira.